src/Discord.Net.Core/Entities/Interactions/Slash Commands/SlashCommandBuilder.cs

```diff
@@ -343,13 +357,20 @@ public string Description
         public ApplicationCommandOptionProperties Build()
         {
             bool isSubType = Type == ApplicationCommandOptionType.SubCommandGroup;
+            bool isIntType = Type == ApplicationCommandOptionType.Integer;
```

Comment:
Should work for `Number` types as well

Reply:
Decimal values are supported on `Number` values, so the check is not needed.

Reply:
Ah alright my bad